lerna.json

```diff
@@ -0,0 +1,9 @@
+{
+  "npmClient": "yarn",
+  "packages": [
```

Comment:
we need to add root to make `cbioportal-frontend` part of the monorepo, all other modules should be added under packages from now on.